Tensile/Contractions.py

```diff
@@ -212,6 +212,9 @@ class ProblemPredicate(Properties.Predicate):
     @classmethod
     def FromOriginalKeyPair(cls, pair):
         (key, value) = pair
+        # TODO - change to use SetConstStrideB
+        if key == 'PackBatchDims':
```

Comment:
I think you want to check `value` here as well - this adds the predicate if this property even exists.